pyqtorch/modules/abstract.py

```diff
@@ -15,21 +16,26 @@ def __init__(self, qubits: ArrayLike, n_qubits: int):
         self.qubits = qubits
         self.n_qubits = n_qubits
 
-    def __mul__(
-        self, other: AbstractGate | pyqtorch.modules.QuantumCircuit
-    ) -> pyqtorch.modules.QuantumCircuit:
+    def __mul__(self, other: AbstractGate | pyq.QuantumCircuit) -> pyq.QuantumCircuit:
```

Comment:
Is there any other `QuantumCircuit` that clashes with this one ? If not, I would import it directly since it is obvious that we are dealing with PyQ types.

Reply:
I tried, but getting circular imports for some reason